### DISCUSSION TITLE
Issues with getting Google approval/verification of the Google Cloud Project for cross platform?

### DISCUSSION THREAD

**Discussion by ROBYER1** [Elringus/UnityGoogleDrive#102] [General]
We followed the setup steps around making a Web and IOS Application type under the same project (this Unity project exports to MacOS + Windows and also Android + IOS) so it is all in one project. The app itself runs fine on all platforms but for external users to login to the Google drive plugin, we needed to get our app approved by Google with OAuth verification for external use.

This is where I am having problems and wondered if anyone else has ran into this scenario?

Response from Google:

"We have noticed that your app is ready for production but there is following development/testing clients in your console:

-(link to the Web Application in the project)
-(link to the IOS Application in the project)

Please update the production client in your console and migrate the development client to separate project in order to proceed with the verification process."

I have responded to their email explaining that it is a cross-platform app built from a single Unity project/codebase and it needs to run on MacOS + Desktop (using Web Application type) and also Android/IOS (using IOS Application type). I have also provided them a link to the setup steps here to show what we are doing https://github.com/Elringus/UnityGoogleDrive#setup-editor-standalones-and-webgl

**Comment by ROBYER1** [comment]
This is the exact approval issue I am having https://serveanswer.com/issue/setting-up-a-google-oauth-production-client

**Reply by DavidJH2** [reply]
Hi @ROBYER1,

I am working on a contract using this API and am trying to decide which route to go.
Have you had any luck with this?

Also, there is a crash with authentication when Google runs it's automatic test.
Have you had this issue recently?

Would Greatly appreciate any feedback!
Thanks
